src/context.ml

```diff
@@ -251,6 +251,30 @@ let create ~(kind : Kind.t) ~path ~base_env ~env_extra ~name ~merlin ~use_findli
   let get_path var = Path.absolute (get var) in
   let stdlib_dir = get_path "standard_library" in
   let natdynlink_supported = Path.exists (Path.relative stdlib_dir "dynlink.cmxa") in
+  let env =
+    (* We redirect the output of all commands, so by default the compiler will disable
+       colors. Since we support colors in the output of commands, we force it via OCAMLPARAM
+       if stderr supports colors. *)
+    if Lazy.force Ansi_color.stderr_supports_colors then
+      let vars =
+        if get "version" <> "4.02.3" then
```

Comment:
We should bind the result of `get "version"` before, so that we don't do `get "version"` twice (here and in the creation of the record)